documentation/modules/exploit/unix/webapp/trixbox_ce_auth_rce.md

```diff
@@ -1,6 +1,6 @@
 ## Introduction
 
-This module exploits a post-authentication OS command injection vulnerability found in Trixbox CE <= v2.8.0.4 which may allow arbitrary command execution on the underlying operating system.
+This module exploits a post-authentication OS command injection vulnerability found in Trixbox CE v2.8.0.4 which may allow arbitrary command execution on the underlying operating system.
```

Comment:
If this affects older versions and you have confirmed this, please revert this change. Only the title of the exploit module needed to be changed.

Reply:
Confirmed reverted.